src/Attachment.tsx

```diff
@@ -33,6 +33,8 @@ const queryString = (query: QueryParams) =>
     .map(key => encodeURIComponent(key) + '=' + encodeURIComponent(query[key].toString()))
     .join('&');
 
+const isNil = (value: any) => value === null || typeof value === "undefined"
```

Comment:
since you're only ever using the negative of this, may I suggest you instead create a function called `exists` which returns `true` if not null or undefined. And you can also use a bit of JavaScript judo by using the `!=` operator:

```ts
const exists = (value: any) => value != null;
```